satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -89,6 +130,14 @@ func rootMutation(log *zap.Logger, service *console.Service, mailService *mailse
 						return nil, err
 					}
 
+					if len(partnerInput) > 1 {
```

Comment:
I think it would be better for readability if you do `if partnerInput != "" { ... }`

Reply:
Good point. Changed it